mentat/terminal/client.py

```diff
@@ -64,15 +64,17 @@ async def _handle_input_requests(self):
         assert isinstance(self.session, Session), "TerminalClient is not running"
         while True:
             input_request_message = await self.session.stream.recv("input_request")
-            # TODO: Make extra kwargs like plain constants
-            if (
-                input_request_message.extra is not None
-                and input_request_message.extra.get("plain")
-            ):
-                prompt_session = self._plain_session
-            else:
-                prompt_session = self._prompt_session
-            user_input = await prompt_session.prompt_async(handle_sigint=False)
+            user_input = None
+            while user_input is None or user_input == "":
```

Comment:
What about when we have a "Y/n" question and the user just hits enter (empty should default to yes)? I also think this would be best to do on the server anyways; we can just put this loop in session.py in the main loop where we wait for user input

Reply:
Yeah that is a lot better